main.go

```diff
@@ -179,16 +179,21 @@ The log file is stored in $BITRISE_DEPLOY_DIR, and its full path is available in
 	log.Infof("Export:")
 
 	args := []string{"xcodebuild", "-showBuildSettings"}
-	if xcworkspace.IsWorkspace(cfg.ProjectPath) {
-		args = append(args, "-workspace", cfg.ProjectPath)
-	} else {
-		args = append(args, "-project", cfg.ProjectPath)
-	}
-	args = append(args, "-scheme", cfg.Scheme)
-	if cfg.Configuration != "" {
-		args = append(args, "-configuration", cfg.Configuration)
+	{
```

Comment:
why is this block in a separate scope?

Reply:
I think it's easier to read this way:
- The scope will result an args array
- In the scope, nothing else happens just the modification of the args array.